packages/mui-core-internal-download-tracker/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@mui/core-internal-download-tracker",
+  "version": "5.10.0",
+  "private": false,
+  "author": "MUI Team",
+  "description": "Internal package to track number of downloads of our design system libraries",
+  "main": "./index.js",
```

Comment:
Are we sure about this one? It doesn't look CommonJS

Reply:
Nothing is ever imported from this package, so it shouldn't break.